src/main/java/com/github/dockerjava/api/DockerClient.java

```diff
@@ -251,6 +256,48 @@
 
     DisconnectFromNetworkCmd disconnectFromNetworkCmd();
 
+    /**
+     * Enables swarm mode for the docker engine and creates a new swarm cluster
+     *
+     * @since 1.24
+     * @param swarmSpec the specification for the swarm
+     * @return the command
+     */
+    InitializeSwarmCmd initializeSwarmCmd(SwarmSpec swarmSpec);
```

Comment:
Oh, the tests are run in parallel? That's complicated even for the swarm tests themselves, because they need to initialize/join/leave swarm mode a few times. How about running each test in its own dind? I added a nice helper method to spin up (and automatically remove) dinds because I need two docker engines in the Join test, anyway.